kafka/errors.py

```diff
@@ -59,7 +59,17 @@ class UnrecognizedBrokerVersion(KafkaError):
 
 
 class CommitFailedError(KafkaError):
-    pass
+    def __init__(self, *args, **kwargs):
+        super(CommitFailedError, self).__init__(
+            """Commit cannot be completed since the group has already
+            rebalanced and assigned the partitions to another member.
+            This means that the time between subsequent calls to poll()
+            was longer than the configured max.poll.interval.ms, which
+            typically implies that the poll loop is spending too much
+            time message processing. You can address this either by
+            increasing the session timeout or by reducing the maximum
+            size of batches returned in poll() with max.poll.records.
```

Comment:
Good call on updating the text. But the CommitFailed error is only raised when using group coordinated groups It will not be seen when using zookeeper offsets from 0.8.2. That said, it can be seen by users that do not have max_poll_interval_ms support (0.9 <= broker < 0.10.1), and so it may need some tweaking when we decide how to manage max_poll_interval_ms in that context.